drift-client/src/main/java/io/airlift/drift/client/address/SimpleAddressSelectorConfig.java

```diff
@@ -38,7 +38,7 @@
         return addresses;
     }
 
-    @Config("addresses")
+    @Config("thrift.client.addresses")
```

Comment:
I'm not sure if this is what we want, as this is a property of address selection, not the Thrift client.